cns/service/main.go

```diff
@@ -880,6 +884,43 @@ func main() {
 		}
 	}
 
+	if cnsconfig.EnableSwiftV2 && cnsconfig.EnableK8sDevicePlugin {
+		initialVnetNICCount := 0
+		initialIBNICCount := 0
+		// Create device plugin manager instance
+		pluginManager := deviceplugin.NewPluginManager(z)
+		pluginManager.AddPlugin(mtv1alpha1.DeviceTypeVnetNIC, initialVnetNICCount)
+		pluginManager.AddPlugin(mtv1alpha1.DeviceTypeInfiniBandNIC, initialIBNICCount)
+
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		// Start device plugin manager in a separate goroutine
+		go func() {
+			for {
```

Comment:
This `for` seems out of place if the intention is to restart `pluginManager` in the event of an error. It might also be wise to have a circuit breaker if it continually fails.

Reply:
added circuit breaker